index.css

```diff
@@ -60,3 +116,29 @@ h1 {
   background-color: #d3d3d3;
   color: #000;
 }
+
+.foot {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
```

Comment:
- [ ] [OPTIONAL] Kindly make sure you space the list of books for a better user experience. 🙏 

![image](https://user-images.githubusercontent.com/74814780/184253508-84022cd0-3db3-48d0-9bb2-600548d7190f.png)